x/bank/types/send_authorization.go

```diff
@@ -34,7 +49,21 @@ func (a SendAuthorization) Accept(ctx sdk.Context, msg sdk.Msg) (authz.AcceptRes
 		return authz.AcceptResponse{Accept: true, Delete: true}, nil
 	}
 
-	return authz.AcceptResponse{Accept: true, Delete: false, Updated: &SendAuthorization{SpendLimit: limitLeft}}, nil
+	isAddrExists := false
+	allowedList := a.GetAllowList()
+
+	for _, addr := range allowedList {
```

Comment:
`allowedList` can be nil. Dumb question maybe: does Go panic if we `range nil`?

Reply:
If `allowList` is nil, it just returns an empty string array. So, it wouldn't panic. I've tested this locally.